_posts/2020-08-05-#19160.md

```diff
@@ -12,20 +12,20 @@ commit: 0602f87
 
 ## Notes
 
-These review club notes are for PR [#19160 multiprocess: Add basic spawn and IPC support](https://github.com/bitcoin/bitcoin/pull/19160) ([branch](https://github.com/ryanofsky/bitcoin/commits/pr/ipc-echo.7)), focused on the second eponymous commit: [`aa4d626`](https://github.com/ryanofsky/bitcoin/commit/aa4d626db1628d5e29572eccfd7fa2d99edeeb60). _IPC_ in the title is short for [Inter-process communication](https://en.wikipedia.org/wiki/Inter-process_communication).
```

Comment:
This first paragraph is mostly redundant. I suggest:

In this review club, we'll focus on the [second commit from this branch](https://github.com/bitcoin-core-review-club/bitcoin/commit/aa4d626db1628d5e29572eccfd7fa2d99edeeb60) (_multiprocess: Add basic spawn and IPC support_). IPC here is short for [Inter-process communication](https://en.wikipedia.org/wiki/Inter-process_communication).

Reply:
Done, thanks.